MPSGameLift/Code/CMakeLists.txt

```diff
@@ -56,14 +56,17 @@ ly_add_target(
         PUBLIC
             AZ::AzCore
             AZ::AzFramework
+            Gem::HttpRequestor
             Gem::Multiplayer.Client
             Gem::LyShine
 
         PRIVATE
             Gem::Multiplayer.Client.Static
             Gem::AWSCore.Static
+            Gem::HttpRequestor.Static
             Gem::AWSGameLift.Client.Static
             Gem::LyShine.Static
+            3rdParty::AWSNativeSDK::Core
```

Comment:
Are you sure this is needed? AWSCore has this already in PUBLIC https://github.com/o3de/o3de/blob/development/Gems/AWSCore/Code/CMakeLists.txt#L34